css/style.css

```diff
@@ -0,0 +1,547 @@
+}
+
+.heading__signin {
+  text-decoration: none;
+  font-weight: 700;
+  color: #000;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  margin-right: 35px;
+}
+
+.heading__cart-wrapper {
+  display: flex;
+  align-items: center;
+  margin-right: 35px;
+}
+
+.heading__cart-link {
+  text-decoration: none;
+  font-weight: 700;
+  color: #000;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  margin-right: 4px;
+}
+
+.heading__cart-img {
+  height: 15px;
+}
+
+.heading__flag {
+  width: 20px;
+}
+
+/* Второй блок */
+
+.main__wrapper {
+  height: 280px;
+  background: url(../img/banner-bg.png);
+  margin-bottom: 54px;
+}
+
+.main__text-wrapper {
+  padding-top: 91px;
+  padding-left: 60px;
+  max-width: 195px;
+}
+
+.main__header {
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #fff;
+  padding-bottom: 15px;
+}
+
+.main__link {
+  position: relative;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.main__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 78px;
+  bottom: 2px;
+  background: url('../icons/arrow.png');
+  width: 45px;
+  height: 8px;
+}
+
+.main__subheader {
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 21px;
+  text-transform: uppercase;
+  color: #000;
+  text-align: center;
+  margin-bottom: 15px;
+}
+
+.main__card-wrapper {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 26px;
+}
+
+.product-card {
+  position: relative;
+  max-width: 230px;
+}
+
+.product-card::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 215px;
+  bottom: 84px;
+  width: 1px;
+  height: 100px;
+  opacity: .2;
+  background: #000;
+}
+
+.product-card:last-child::after {
+  display: none;
+}
+
+.product-card__img {
+  padding-bottom: 23px;
+}
+
+.product-card__discr {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 11px;
+  color: #000;
+  text-align: left;
+  padding-bottom: 5px;
+}
+
+.product-card__options {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__price {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 13px;
+  color: #000;
+}
+
+.product-card__sel {
+  width: 50px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.product-card__btn {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  border: none;
+}
+
+.product-card__btn_selected {
+  height: 9px;
+  width: 9px;
+}
+
+.product-card__btn_grey {
+  background: #c4c4c4;
+}
+
+.product-card__btn_blue {
+  background: #41479b;
+}
+
+.product-card__btn_red {
+  background: #b62e3c;
+}
+
+.product-card__btn_yellow {
+  background: #fdb431;
+}
+
+.product-card__btn_brown {
+  background: #762e31;
+}
+
+.product-card__btn_orange {
+  background: #e29d69;
+}
+
+.product-card__btn_black {
+  background: #2c2c2c;  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)